releases/release-1.21/release-notes/sessions/maps-1611771700.json

```diff
@@ -0,0 +1 @@
7f5cb0dddd9eabfebfc5ed974bddeea7dc4"},{"nr":98436,"hash":"d9175c9e45cd918d3b8bf5d1f19a17fb7528434d"},{"nr":98138,"hash":"74ac0a2b9401b2bc5cbd39b33fb649a0ab8e7a18"},{"nr":97366,"hash":"c20328efda4a64c378fcb69e201ebd7eae523704"},{"nr":97635,"hash":"a348e5a13a6a5866fc93b5ed8248d9a14216bbb0"},{"nr":97941,"hash":"f0443fc2c43c4453e45aefb0841dd14fb1da069d"},{"nr":97693,"hash":"8989aa051fad8a64dd967c4fd9117f0e47b0625e"},{"nr":98252,"hash":"503beae3a5f7d19ffbc283baa983aa6286c830f2"},{"nr":97764,"hash":"0edd935237dd1e6adf536f4ba6847e581772e52d"},{"nr":98401,"hash":"550dc732049ffb309abc77686c0cee6353debfcf"},{"nr":97413,"hash":"54b00c8f9679feaebd2d76225eea9e1f79794e6c"},{"nr":96573,"hash":"a97876e0602ac0479c70ff1a11d6632e2be697f2"},{"nr":96675,"hash":"f0c743e4d82e0ef17e1ca26868067a24b7a2f7c4"},{"nr":97141,"hash":"26cb3ff81bdbf707e052268e7c1e402f3fb5e07e"},{"nr":98158,"hash":"900a9351c36504b21eb81b85edd0502ecd5cdf69"},{"nr":98312,"hash":"79d726f90ab0b0538255a6ae8f56549591688114"},{"nr":96552,"hash":"e2d7d9576d8ff1f093599e1b780b3426141ad6cd"},{"nr":98200,"hash":"b382fe0eb15c04594265b57be0a9a2117a0e9c17"},{"nr":98093,"hash":"9173ed0c6d6ff6194b2fa7893cba84a093ff87dc"},{"nr":97480,"hash":"5c17779ed8d752c3054840de0eb3d8b3e4b6e7db"},{"nr":98169,"hash":"282b5f03a8cf4475eba7512be5b6d95a7ff89b96"},{"nr":98134,"hash":"bbf46e46f2f6b5f08466141a96302673970a8fa9"},{"nr":96844,"hash":"5332708001695524516e5fac6696ddc3622e6ecd"},{"nr":97919,"hash":"5b8c1db64ff94e5c97398ccb911a4d2624302387"},{"nr":92938,"hash":"5f966a136c0ee9d294d4d2d0e77bcca0bc4b5f62"},{"nr":97946,"hash":"4f1f725d75ec60bc93d82c62430765f8e720f0ee"},{"nr":97618,"hash":"76f63a5f5ef768ef0b37841a4679bbcd42f14120"},{"nr":97543,"hash":"370b850fd4eb18b3579aed52d233540d68401c05"},{"nr":98111,"hash":"13fe5af443e325241b6dcfc24768bd4ddc939dcc"},{"nr":97967,"hash":"5a1f718b5d36560a3458b260b258e2df06285c48"},{"nr":97678,"hash":"4b1414ef8066bb464f6db7ed7b54c4b4f14782d1"},{"nr":96561,"hash":"789a012cfe2d60b2eccf9cde3774a1f0453b3e9f"},{"nr":98088,"hash":"308fabe754eba59ad0ccbe599021094093d9da8c"},{"nr":93510,"hash":"1984b1e1261c5c0eec0e6889fbcb00f97025eeb5"},{"nr":97877,"hash":"f6c9e317a8576ceeb3104f2f4fe4d2539f5a5998"},{"nr":94858,"hash":"f4ce3222b58c541fef5af2ecbe0649e16246b9c8"},{"nr":98430,"hash":"8a6f6185c606fd3f6a6db798f885636cb430ae2d"},{"nr":95111,"hash":"bc7fd3bdd173d7c78ee1cb4920f06e9484baf90c"},{"nr":97492,"hash":"12b0bf49cb2363e19bf5f1a5884452e52bb6340c"},{"nr":95664,"hash":"80e8ca6511ca11c558d4c2dfeaf683f15a83f0f0"},{"nr":98103,"hash":"7ea828d60a47a1ef0083bc9120a318cb265b5292"},{"nr":61595,"hash":"a4a1c012cf1ecfb0604ecb92bc1f1cd7264f5263"},{"nr":97115,"hash":"54c3a17d7e027fb11011161866fe7c0a18618cd6"},{"nr":98284,"hash":"782a9522dfc71ec6419995854eeff11a8adf2ce1"},{"nr":96617,"hash":"908c42053175ed4f1cc2e47ad39500ce5c50c278"},{"nr":98068,"hash":"d8e614dbdf163f4fe6a3b7c8bd589c6c0148c30a"},{"nr":97922,"hash":"282f421285571406097cb38c14a68e8c2443c1e7"},{"nr":93179,"hash":"517140306bfa563358f81f2aa655c0881fffbf1f"}]}
```

Comment:
did you mean to check this in? I recall that you don't want to review them on the first PR.

Reply:
in other words, if you're submitting this PR as "raw data" / unreviewed content, then please exclude this file from being checked in

Reply:
@wilsonehusin this PR was automatically generated so this file was included by default!

Reply:
Right, I'm guessing that this is because you went through the notes and when asked "Do you want to modify the notes?" you answered "No" on each.

To "submit without reviewing", the right way to do it (for now) is by hitting Ctrl+C on the first note that it displayed.

So if you didn't review the content, please remove this file from being committed. That way, the next person running release notes can go through the ones added between alpha.1 and alpha.2, without going through the ones I checked in.

Does that make sense?

Reply:
Thanks for the review, PR updated, let me know your thoughts @wilsonehusin :)